openmetadata-ui/src/main/resources/ui/src/components/ExploreV1/exploreV1.less

```diff
@@ -64,6 +64,10 @@
   .filters-row {
     border-bottom: 1px solid @border-color;
   }
+
+  .alert-link.text-xs {
```

Comment:
Can't you use .text-xs it has font-size: 12px;

Reply:
".ant-typography a"  defined in font.less have more specificity over it, so it was overriding by that css if directly applying text-xs

Reply:
fixed: inheriting size rather then specifying statically